tests/pds/pds_test.go

```diff
@@ -0,0 +1,78 @@
+package tests
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/torpedo/tests"
+
+	"github.com/portworx/torpedo/pkg/pds"
+)
+
+func TestPDS(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Torpedo: PDS")
+}
+
+var (
+	controlPlane  *pds.ControlPlane
+	targetCluster *pds.TargetCluster
+)
+
+var _ = BeforeSuite(func() {
+	cpKubeconfig, err := tests.GetSourceClusterConfigPath()
```

Comment:
@aghodke1312 Let me know if you wanna change the way we pass the kubeconfigs to the test.
Reusing this helper assumes there are two comma-separated paths in the `KUBECONFIGS` env variable (first being the control plane, second being the target cluster in our case).